### DISCUSSION TITLE
ea_giving_barriers Discussions: free form starting place

### DISCUSSION THREAD

**Discussion by daaronr** [daaronr/ea_giving_barriers#17] [General]
## Discussions: Free form, 'introduce yourself, comments Q&A'' (for now

Note we also have 2 relevant slack channels .... but this 'share document' format seems like it can be more helpful. 

`EA Psych...`:  The key channel is `ea_litsurvey_ppr`.
`charitablegivingexperiments`: Key channel is ea_giving_barriers-synthesis

If you like gh suggests "comment below with an introduction of yourself and tell us about what you do with this community."

Nice: 
>   ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by daaronr** [comment]
mjusten@wisc.edu (Michel) asked a series of questions, I/we try to answer these below, may expand later 

## "Questions about project" 

**('substance ... first part of his questions')**

1. *Where is this progress in relation to its end goal? For example, is the organization presented in the bookdown the final organization just waiting for more content to be filled in, or are you still considering other outlines?*

It's an ongoing Kaizen thing, continually improved. 

- I think we are well along in the outline, and it has some nice overlaps with Caviola et al's 2021 organization. 
- On the other hand it is still open to improvement/suggestions/adjustment, and we have indeed made some adjustments in recent months

2.  *I am familiar with psychological research on effective giving because of my thesis literature review, so that's where I'll look to contribute. But I should flag that I have very little economics experience and my statistics/knowledge about meta-analyses is shaky as an undergraduate.*

That's fine. The project needs a range of skills and contributions, and the one you propose to make is a very important one.

**Comment by daaronr** [comment]
## Questions  about contributing to the project [tech/setup etc] 

...  from mjusten@wisc.edu (Michel)

1. *Currently when I access the repo on GitHub desktop it (comfortingly) states that I don't have write access, and my only option is to fork. Could you give me write access? From there I can create my own branch, which, as you guessed in your previous email, I would prefer to work on instead of the master.... The last thing I want to do is mess anything existing up, so I'll familiarize myself more with git first.*

Done. 


2. *I was a little confused on the citation section, which appears to be complicated by the new RStudio update removing Citr (I see this is an open issue on your bookdown template repo). At your convenience, some updated instructions on this could be helpful, but I'll also look into the RStudio update and a BibTeX for Zotero tutorial to see if I can figure it out. *

Yes, the new Rstudio makes this process 'easier' but it means our old Citr system is obsolete. I think we have updated the instructions at bottom #1  #, but perhaps we need to clarify further? A key challenge is knowing 'where the citations are put when you add them' and 'which bib file is being called.;

3. *Could you give me access to the Zotero shared reference library? I applied to join it here so hopefully you got a notification, otherwise my Zotero account is associated with this emailed*

Just added you


4. *What is the final bibliography file? From exploring the repo and index.Rmd I get the sense that it is "Reference.bib", but section in the bookdown appendix state, “the bibliography for this project is giving_keywords.bib.” as well as ”Make sure ‘add references to’ specifies the “…ea_giving_barriers/support/reinstein_bibtex.bib” folder." Are those lines potentially outdated?*

Yes, potentially outdated. But we should check and doublecheck this and then make it clear exactly where the references are put. (@oskasf ?)

4. *How should I use hypothesis for this project? I flagged two links that weren't working in the appendix, but not sure if there is a better way I should communicate something like that.*

Hypothes.is as I use it does not 'callout' when a comment has been made, I have to search for it. So please email or Slack me to let me know ... and I'll try to check in periodically. (Note it should email you when someone responds to your comment). 

A good alternative, especially for something important/recurring, is to raise an `Issue` within this repo.

> I learned a lot in the appendix and the related resources. I was familiar with Zotero and .Rmd format beforehand, but I didn't have much experience with Git(Hub) and Bookdowns. The instructions and four videos linked were very helpful!)

That's great!

**Comment by MichelJusten** [comment]
Great, thanks for answering these! Following the gh prompt to introduce myself: I am an incoming senior undergraduate at the University of Wisconsin-Madison studying Psychology and Neurobiology. I'm also the co-founder/president of the EA student org there, and collectively those explain my interests pretty well: I am drawn to EA-aligned/potentially impactful psychology research. I am currently working on a senior thesis examining (effective) giving in individuals with damage to their ventromedial prefrontal cortex. I hypothesize that impaired instinctive emotions, which is a consequence of damage to this region, might make people donate 1) less but 2) more effectively when there is a conflict between emotional appeal and charity's effectiveness. I'm always open to talk to folks about this!
